README.md

```diff
@@ -0,0 +1,20 @@
+A package for integrating PennyLane with Orquestra.
```

Comment:
I imagine the readme will be updated to match those of the other plugins in a future PR? (the other plugins use ReST)

Reply:
Yep, this is just a temporary one, for now, will be updated as part of adding the docs.